sos/policies/init_systems/pebble.py

```diff
@@ -0,0 +1,53 @@
+# Copyright (C) 2024 Canonical Ltd., Arif Ali <arif.ali@canonical.com>
+
+# This file is part of the sos project: https://github.com/sosreport/sos
+#
+# This copyrighted material is made available to anyone wishing to use,
+# modify, copy, or redistribute it subject to the terms and conditions of
+# version 2 of the GNU General Public License.
+#
+# See the LICENSE file in the source distribution for further information.
+
+from sos.policies.init_systems import InitSystem
+from sos.utilities import shell_out
+
+
+class PebbleInit(InitSystem):
+    """InitSystem abstraction for Pebble systems"""
+
+    def __init__(self, chroot=None):
+        super(PebbleInit, self).__init__(
+            init_cmd='pebble',
+            list_cmd='services',
+            query_cmd='pebble services',
+            chroot=chroot
+        )
+        self.load_all_services()
+
+    def parse_query(self, output):
+        for line in output.splitlines()[1:]:
+            if line.split()[2] == 'active':
+                return 'active'
+        return 'unknown'
+
+    def load_all_services(self):
+        svcs = shell_out(self.list_cmd, chroot=self.chroot).splitlines()[1:]
+        for line in svcs:
+            try:
+                name = line.split()[0]
+                config = line.split()[1]
+                self.services[name] = {
+                    'name': name,
+                    'config': config,
+                }
+            except IndexError:
+                pass
```

Comment:
To keep CodeQL happy, let's add a comment here explaining either how we got to this exception (header/foooter output?) or some similar note to future-us.

Reply:
sure, it was a copy-paste from the systemd one. So, I think we ought to fix tat one too ;)